packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -50,7 +53,7 @@ const HELLO_WORLD = 'hello-world'
 const TEST_PROJECT = 'test-project' // TODO: This will be replaced with the `isomorphic-client` config.
 const PROMPT = 'prompt'
 
-const PRESETS = [TEST_PROJECT, PROMPT, HELLO_WORLD]
+const PRESETS = [TEST_PROJECT, PROMPT, HELLO_WORLD, HELLO_WORLD_TEST_PROJECT]
```

Comment:
This was an oversight on the previous PR – it's unrelated, but we should ship it.

Reply:
cool, i tested the hello world presets and both `HELLO_WORLD` and `HELLO_WORLD_TEST_PROJECT` works...